package.json

```diff
@@ -35,7 +35,14 @@
     "build:node": "npm run build:node:scripts && npm run build:node:views",
     "build": "npm run clean && npm run build:static && npm run build:node",
     "serve": "npm run build && node scripts/serve.js",
+    "test:js": "karma start",
```

Comment:
This requires karma to be installed globally on the user's machine. We should get it from `./node_modules/bin/` instead to avoid that. See e.g. the `lint` rule.

Reply:
Right!